cmd/shortener/main.go

```diff
@@ -1,3 +1,17 @@
 package main
 
-func main() {}
+import (
+	"github.com/KorsakovPV/shortener/cmd/shortener/api"
+	"github.com/KorsakovPV/shortener/cmd/shortener/config"
+	"log"
+	"net/http"
+)
+
+func main() {
+
+	config.ParseFlags()
+
+	log.Printf("Shortener start on %s. Default base URL %s.", config.Config.FlagRunAddr, config.Config.FlagBaseURLAddr)
+
+	log.Fatal(http.ListenAndServe(config.Config.FlagRunAddr, api.Router()))
```

Comment:
```
	err := http.ListenAndServe(":8080", http.HandlerFunc(handler))
	if err != nil {
		log.Fatal(err)
	}
```


```
	log.Fatal(nil)
```
When this code is executed, it will panic at runtime with the error message "log.Fatal: nil". The panic will cause the program to terminate abruptly.

To avoid this error, it is recommended to provide a non-nil error or message argument to `log.Fatal()`, which helps to indicate the reason for the fatal error and provide useful information for debugging.

Reply:
ok. corrected.